cookbook/getTimeZoneObjectFromIanaName.js

```diff
@@ -0,0 +1,11 @@
+/**
+ *
+ * @param ianaTimeZoneName {string} TimeZone String
+ * @returns {Temporal.Timezone}
+ * @example getTimeZoneObjectFromIanaName("Europe/London")
+ */
+function getTimeZoneObjectFromIanaName(ianaTimeZoneName) {
+    return new Temporal.TimeZone(ianaTimeZoneName);
+}
+
+getTimeZoneObjectFromIanaName("Europe/London").toString(); // Europe/London
```

Comment:
Maybe it would be useful to show how to use some of the methods of the `TimeZone` objects here.

In any case, please rebase and add the new test to `docs/cookbook/all.mjs`